### PR TITLE
 Prepare to release 1.1.1

### PR DESCRIPTION
**Pull request by ethan-schaffer**
Addressing https://github.com/studiopress/seo-data-transporter/issues/17#event-2472470052

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "description": "SEO Data Transporter helps you transfer post/page specific SEO data, like custom doctitles, custom META descriptions and keywords, etc., from one platform (theme or plugin) to another.",
     "author": "StudioPress",
     "authoruri": "http://www.studiopress.com/",
```
Comment (marksabbath) [inline package.json]: @ethan-schaffer would you mind updating this adding the `s` here?

```suggestion
    "authoruri": "https://www.studiopress.com/",
```

**Review by marksabbath — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
...
-    "version": "1.1.0",
+    "version": "1.1.1",
     "license": "GPL-2.0+",
```
Comment (dreamwhisper) [inline package.json]:
```suggestion
    "license": "GPL-2.0-or-later",
```

File: package.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
...
     "licenseuri": "http://www.gnu.org/licenses/gpl-2.0.html",
```
Comment (dreamwhisper) [inline package.json]:
```suggestion
    "licenseuri": "https://www.gnu.org/licenses/gpl-2.0.html",
```

**Review by dreamwhisper — APPROVED** [review]
Approved, with non-blocking suggestions.

**Comment by dreamwhisper** [comment]
@ethan-schaffer does this need a .pot language file update?

**Comment by ethan-schaffer** [comment]
@dreamwhisper It might, but I wasn't able to use npm to make an update, as the package.json file doesn't have any scripts declared.

**Pull request merged**